### DISCUSSION TITLE
Suggestion: remove `tvg-name` attribute from channel descriptions

### DISCUSSION THREAD

**Discussion by freearhey** [iptv-org/community#519] [Ideas]
I couldn't find any application that uses this attribute in any way (correct me if I'm wrong), so I think we can remove it from playlists completely.

**Comment by jkemp814** [comment]
https://kodi.wiki/view/Add-on:PVR_IPTV_Simple_Client

tvg-id
    Corresponds to 'channel-id' field in the EPG xml file.
    If missing from the M3U file, the addon will use the 'tvg-name' tag to map the channel to the EPG.
tvg-name
    Corresponds to 'display-name' field in the EPG xml file. Space characters need to be replaced with '_' (underscore character).
    If missing from the M3U file, the addon will use the 'channel-name' tag to map the channel to the EPG.

**Reply by jkemp814** [reply]
We always have a tvg-id field, so it shouldn't fail.

**Reply by freearhey** [reply]
Exactly.
